.github/workflows/verify.yml

```diff
@@ -1,4 +1,4 @@
-name: Verify Submodule Pull Request
+name: Verify OSOCE Submodule Pull Request
```

Comment:
This change is not needed, the workflow is compatible with any repo. OSOCE is only the default option for `inputs.repo`.